cap-ci/jobs/minibroker-integration-tests.tmpl

```diff
@@ -0,0 +1,57 @@
+{{ define "jobs_minibroker-integration-tests" }}
+- name: minibroker-integration-tests-{{ .scheduler }}-{{ .backend }}-{{ .avail }}
+  serial_groups: [{{ .scheduler }}-{{ .backend }}-{{ .avail }}]
+  public: false
+  plan:
+  - get: catapult
+  - get: minibroker-integration-tests
+  - get: s3.minibroker
+  - get: mits-gh-release
+  - get: {{ .backend }}-pool.kube-hosts
+    {{- if ne .position 0 }}
+    passed:
+    - {{ index .jobs_enabled (sub .position 1) }}-{{ .scheduler }}-{{ .backend }}-{{ .avail }}
+    {{- end }}
+    trigger: true
+  - task: test-{{ .scheduler }}
+    {{- if has .workertags .backend }}
+    tags: [{{ index .workertags .backend }}]
+    {{- end }}
+    timeout: 1h30m
+    input_mapping:
+      pool.kube-hosts: {{ .backend }}-pool.kube-hosts
+    config:
+      platform: linux
+      image_resource:
+        type: registry-image
+        source:
+          repository: splatform/catapult
+      inputs:
+      - name: catapult
+      - name: minibroker-integration-tests
+      - name: pool.kube-hosts
+      - name: s3.minibroker
+      - name: mits-gh-release
+      params:
+        QUIET_OUTPUT: true
+        DOWNLOAD_CATAPULT_DEPS: false
+        ENABLE_EIRINI: {{ eq .scheduler "eirini" }}
+{{- if eq .avail "ha" }}
+        HA: true
+{{- end }}
+{{- if eq .avail "all" }}
+{{- print .allbells }}
```

Comment:
Some keys are unreadable... `allbells`? What's it supposed to mean?

Reply:
yeah, thats legacy. its for autoscaler. Will need separate refactoring for cap-ci and catapult.
eg: https://github.com/SUSE/cap/blob/master/cap-ci/jobs/cf-acceptance-tests.tmpl#L37

Reply:
Are there cards/tickets/issues opened to fix this? If yes, could you link them here (at least the relevant one for this specific case, or if there's only one generic, it's good to cross-reference so we don't miss this specific case)?

Reply:
created a Jira for it